packages/next-extraction-plugin/package.json

```diff
@@ -2,10 +2,19 @@
   "name": "@griffel/next-extraction-plugin",
   "version": "0.0.1",
   "private": true,
-  "description": "TBD",
+  "description": "NextJS plugin that add Griffel's CSS extraction to webpack loaders pipeline",
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "https://github.com/microsoft/griffel"
+  },
+  "dependencies": {
+    "browserslist": "^4.19.1",
+    "next": "^12.0.5"
```

Comment:
Let's move `next` to `peerDependencies` as it's not expected to have multiple versions of Next.js installed